cloudevents/extensions/expirytime.md

```diff
@@ -0,0 +1,58 @@
+# Expiry Time Extension
+
+This extension provides a mechanism to hint to [consumers](../spec.md#consumer)
+or [intermediaries](../spec.md#intermediary) a time after which an
+[event](../spec.md#event) may be ignored.
+
+In distributed systems with message delivery guarantees, events may be delivered
+to a consumer some significant amount of time after an event has been sent via
```

Comment:
perhaps stop after "sent" since this would apply whether an intermediary is involved or not, right?